packages/components/src/utils/getNextFocus.ts

```diff
@@ -24,30 +24,39 @@
 
  */
 
-const getTabStops = (ref: HTMLElement): HTMLElement[] =>
-  Array.from(ref.querySelectorAll('a,button:not(:disabled),[tabindex="0"]'))
-
-export const moveFocus = (
-  direction: number,
-  initial: number,
-  element?: HTMLElement | null
-) => {
-  if (element) {
-    const tabStops = getTabStops(element)
+export const getTabStops = (ref: HTMLElement): HTMLElement[] =>
+  Array.from(
+    ref.querySelectorAll(
+      'a,button:not(:disabled),[tabindex="0"],[tabindex="-1"]:not(:disabled)'
+    )
+  )
+
+/**
+ * Returns the next focusable inside an element in a given direction
+ * @param direction 1 for forward -1 for reverse
+ * @param element the container element
+ */
+export const getNextFocus = (direction: 1 | -1, element: HTMLElement) => {
```

Comment:
```suggestion
export const getNextFocus = (direction: 1 | -1, element: HTMLElement) => {
```

Might be worth it to make direction take string "forward" or "reverse" (I'd argue it's a little more readable when calling the function) but I'll leave it up to preference!